src/test/java/com/google/cloud/spanner/jdbc/it/ITJdbcSimpleStatementsTest.java

```diff
@@ -59,14 +65,24 @@ public static List<DialectTestParameter> data() {
 
   @Parameter public DialectTestParameter dialect;
 
+  private Database database;
+
+  @Before
+  public void setup() {
```

Comment:
Should the setup method go into `ITAbstractJdbcTest` class?

Reply:
I decided to move that away from the abstract base class, as it was one of the main reasons that things were not working properly. The data model that is created by the different test classes is different, and that went wrong for the parameterized test classes, as they need to take the dialect into account as well. So this solves a lot of hassle for those tests at the expense of a little code duplication.